projects/ngrx-auto-entity/src/util/func.ts

```diff
@@ -1,4 +1,4 @@
-export const pipe = (...fns) => data => fns.reduce((value, fn) => fn(value), data);
+export const compose = (...fns) => data => fns.reduce((value, fn) => fn(value), data);
```

Comment:
The change was to avoid conflict with the `pipe` function that comes with RxJs. Composition of functions is what pipe does, and compose has long been an alternative name. Usually, with compose, the second function is actually defined first, but this is an internal-only function and not exposed to consumers, and I really just needed a different name for it than `pipe`.